### ISSUE TITLE
Clarifications recommendation around exchange of location data

### ISSUE THREAD

**Issue by ldodds** [alphagov/tech-and-data-standards#48]
I have some questions around the recommendations specified at:

https://www.gov.uk/government/publications/open-standards-for-government/exchange-of-location-point

The document says that:

* for the geographical scope of ETRS891, ETRS89 must be used, WGS 84 may also be used
* other coordinate systems may be used in addition to WGS 84 and ETRS89

What does this actually mean in practice in terms of specifying conformance criteria and designing data formats? For example, within the scopre of ETRS891, should a data file include points with a ETRS89 CRS and then may, in addition, include the points in other CRS? Or is there a choice? 

The section on Functional Needs in the guidance doesn't really elaborate. In fact it makes a case for using WGS84.

As a concrete example, the newly published [Brownfield Land Register standard](https://www.gov.uk/government/uploads/system/uploads/attachment_data/file/633593/BrownfieldLandRegisters-DataStandard.pdf) says that local authorities should use ETRS89, but the standard allows points to be specified in in other CRS systems. 

I was just at a workshop on this standard where was some debate about the utility of ETRS89. E.g. local authority systems may not store this natively and consumers of the open data are perhaps more likely to want WGS84.

There also seems to be some inconsistency in section 5, which says:

"applications that consume data sets containing points must promote and prefer WGS 84".

Promote and prefer WGS 84 seems at odds with requiring use of ETRS89?

I understand that ETRS89 is the standard CRS used in the EU, so can see why it has been referenced. But I think it might be useful to clarify some of the intended outcomes here.

**Comment by edent** [comment]
tl;dr - the [EU's INSPIRE Directive](https://inspire.ec.europa.eu/id/document/tg/rs) compels us to use ETRS89 for European things.

Data files should specify which CRS they are using. If they are EU based, ETRS89 *must* be used. Any other CRS *may* be used if you want.

For non-EU stuff, use and specify WSG86.  You may also add in other CRS if you want.

## Background

The panel which considered this proposal said:

> Panel members noted that the UK is compelled by the INSPIRE Directive to use the ETRS89 standard in certain spatial datasets.  If locations in Europe were being described the proposal, as it stands, would mean that two coordinates would have to be published. It was agreed that this was not ideal. It was noted that for many uses which of the two coordinates being used would not be critical as they differ by only 60cm. In addition, which coordinate system was being used would have to be recorded with the data. For those requiring accuracy there are transformation registries available. The Panel agreed that information should be added to the proposal to advise describing the standard being used where possible.

As recorded in [the meeting minutes](https://github.com/alphagov/open-standards/blob/master/docs/_meetings/2015%E2%80%9010%E2%80%9023-data-panel-meeting.md)

The Open Standards Board said:

> For the geographical scope of the European Terrestrial Reference System 19891 (ETRS89), ETRS89 must be used. However, for many use cases where the effect of continental drift is not considered significant the United States Government Department of Defense (DoD) World Geodetic System 1984 (WGS 84) is functionally equivalent to ETRS89. Therefore, for the rest of the world, WGS 84 must be used.

> The board recommended that other standards may be used where appropriate, for example when greater accuracy is required in a local survey by engineers.

> The board notes that it is possible to map other coordinate systems to the standards chosen.

See the [Board Minutes](https://github.com/alphagov/open-standards/blob/master/docs/_meetings/2015%E2%80%9011%E2%80%9013-open-standards-board.md)

**Comment by ldodds** [comment]
That's a really helpful clarification. I think it would be useful to include something to that effect in the recommendation text. The need to specify two sets of coords was something I was wrestling with.

A related, but separate concern was I can't use GeoJSON and comply with this to publish official EU data, as it [only supports WGS84](https://tools.ietf.org/html/rfc7946#page-12). But guess that's an issue for the EU.

**Comment by edent** [comment]
Agreed. @Lawrence-G @RosalieDM please can we update the wording to make it a little clearer.

I'll leave this issue open until it is fixed - in case anyone else wants to provide more information.

**Comment by TechPunk316** [comment]
Hi @ldodds - thank you for your feedback. We have now updated our guidance to make it clearer.

https://www.gov.uk/government/publications/open-standards-for-government/exchange-of-location-point
